sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -557,14 +557,141 @@ public async Task ValidateMergeCertificate()
+        {
+            var contact1 = new CertificateContact
+            {
+                Name = Recording.GenerateId(),
+                Phone = Recording.GenerateId(),
+                Email = "test@microsoft.com"
+            };
+
+            var contact2 = new CertificateContact
+            {
+                Name = Recording.GenerateId(),
+                Phone = Recording.GenerateId(),
+                Email = "test1@microsoft.com"
+            };
+
+            var contacts = new List<CertificateContact>
+            {
+                    contact1,
+                    contact2
+            };
+
+            RegisterForCleanUpContacts(contacts);
+
+            await Client.SetContactsAsync(contacts);
+            Response<IList<CertificateContact>> getContactsResponse = await Client.GetContactsAsync();
+
+            IList<CertificateContact> getContacts = getContactsResponse.Value;
+
+            Assert.NotNull(getContacts);
+            Assert.AreEqual(2, getContacts.Count);
+            Assert.IsTrue(getContacts.Equals(contacts));
+            Assert.IsTrue(getContacts.Contains(contact1));
+            Assert.IsTrue(getContacts.Contains(contact2));
+        }
+
+        [Test]
+        public async Task VerifyGetCertificatePolicy()
+        {
+            string certName = Recording.GenerateId();
+
+            CertificatePolicy certificatePolicy = DefaultPolicy;
+
+            await Client.StartCreateCertificateAsync(certName, certificatePolicy);
+
+            RegisterForCleanup(certName);
+
+            Response<CertificatePolicy> policyResponse = await Client.GetCertificatePolicyAsync(certName);
+
+            CertificatePolicy policy = policyResponse.Value;
+
+            Assert.NotNull(policy);
+            Assert.AreEqual(DefaultPolicy.KeyType, policy.KeyType);
+            Assert.AreEqual(DefaultPolicy.IssuerName, policy.IssuerName);
+            Assert.AreEqual(DefaultPolicy.ReuseKey, policy.ReuseKey);
+        }
+
+        [Test]
+        public async Task VerifyUpdateCertificatePolicy()
+        {
+            string certName = Recording.GenerateId();
+
+            CertificatePolicy certificatePolicy = DefaultPolicy;
+
+            await Client.StartCreateCertificateAsync(certName, certificatePolicy);
+
+            RegisterForCleanup(certName);
+
+            certificatePolicy = new CertificatePolicy
+            {
+                Subject = "CN=mydomain",
+                ReuseKey = true,
+                Exportable = false
+            };
+
+            Response<CertificatePolicy> updatePolicyResponse = await Client.UpdateCertificatePolicyAsync(certName, certificatePolicy);
+
+            CertificatePolicy updatePolicy = updatePolicyResponse.Value;
+
+            Assert.NotNull(updatePolicy);
+            Assert.NotNull(updatePolicy.UpdatedOn);
+            Assert.AreEqual(certificatePolicy.Subject, updatePolicy.Subject);
+            Assert.AreEqual(certificatePolicy.ReuseKey, updatePolicy.ReuseKey);
```

Comment:
The test failed because UpdatePolicy method cannot update **Reusekey** and **Exportable** property. Is this by design ?

Reply:
For one, you need to wait for the operation returned from `StartCreateCertificateAsync` to finish. It returns a `CertificateOperation` you must await. See above for examples.

Second, you can't change exportable to `true` afterward - I know that for sure. It's possible you can't just change it period. @schaabs would know for sure from the backend side of things.